src/coreclr/src/jit/compiler.cpp

```diff
@@ -876,6 +930,8 @@ var_types Compiler::getArgTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
 //
 // Arguments:
 //    clsHnd         - the handle for the struct type
+//    callConv       - the calling convention of the function
```

Comment:
with adding of `GTF_CALL_M_UNMGD_INST_CALL` do we have injection between `CorInfoUnmanagedCallConv ` and `GTF_CALL_M_*` flags? Can we call `getReturnTypeForStruct` when we have a `GenTreeCall*` node but don't have `CorInfoUnmanagedCallConv `?

Reply:
No, once we have a GenTreeCall we always have the unmanaged calling convention. I'll try to remove that extra flag and instead use a helper function.